src/data/types/capital-types.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  AuthorityType,
+  LoanProgramStatus,
+  ProjectType,
+} from '@rewiringamerica/capital-common';
+import { FromSchema } from 'json-schema-to-ts';
+
+/**
+ * Schema for Image.
+ */
+export const IMAGE_SCHEMA = {
+  $id: 'Image.schema.json',
+  type: 'object',
+  properties: {
+    src: { type: 'string' },
+  },
+  required: ['src'],
+  additionalProperties: false,
+} as const;
+
+/**
+ * Schema for EligibleProjectType.
+ */
+export const ELIGIBLE_PROJECT_TYPE_SCHEMA = {
+  $id: 'EligibleProjectType.schema.json',
+  type: 'object',
+  properties: {
+    type: { type: 'string', enum: Object.values(ProjectType) },
+  },
+  required: ['type'],
+  additionalProperties: false,
+} as const;
+
+/**
+ * Schema for FinancialAuthority.
+ */
+export const FINANCIAL_AUTHORITY_SCHEMA = {
+  $id: 'FinancialAuthority.schema.json',
+  type: 'object',
+  properties: {
+    id: { type: 'number' },
+    authority_type: { type: 'string', enum: Object.values(AuthorityType) },
+    name: { type: 'string' },
+    description: { type: ['string', 'null'] },
```

Comment:
I gotta admit I didn't know `type` could be an array until seeing this. That could have saved me a lot of trouble in the early days :/